### PR TITLE
add physical package object to definitions

### PR DESCRIPTION
**Pull request by rsnancollas**
package definition draft for passives as well as generally for all components

### CODE REVIEW

File: part-spec/definitions.json [hunk 1]
```diff
@@ -346,5 +346,41 @@
...
+        "description": "height of a package",
+        "type": "number"
+      },
+      "dimensionUnit": {
+        "description": "unit used to describe package dimensions",
+        "enum": [
+          "mils",
+          "milimeter",
+          "inches",
+          "centimeter"
```
Comment (rsnancollas) [inline part-spec/definitions.json]: remove centimeter

File: part-spec/definitions.json [hunk 2]
```diff
@@ -346,5 +346,41 @@
...
+      "height": {
+        "description": "height of a package",
+        "type": "number"
+      },
+      "dimensionUnit": {
+        "description": "unit used to describe package dimensions",
+        "enum": [
+          "mils",
+          "milimeter",
+          "inches",
```
Comment (rsnancollas) [inline part-spec/definitions.json]: maybe remove inches

File: part-spec/definitions.json [hunk 3]
```diff
@@ -346,5 +346,41 @@
...
+          "centimeter"
+        ]
+      },
+      "standardPackageType": {
```
Comment (rsnancollas) [inline part-spec/definitions.json]: standardPackageSize

File: part-spec/definitions.json [hunk 4]
```diff
@@ -346,5 +346,41 @@
...
+        "description": "name of standard package type (imperial)",
+        "enum": [
+          "0201",
+          "0402",
+          "0603",
+          "0805",
+          "1206"
+        ]
```
Comment (rsnancollas) [inline part-spec/definitions.json:380]: standardPackageType:

examples:
BGA
TSSOP

File: part-spec/definitions.json [hunk 5]
```diff
@@ -346,5 +346,41 @@
...
+        "enum": [
+          "mils",
+          "milimeter",
+          "inches",
+          "centimeter"
+        ]
+      },
+      "standardPackageType": {
+        "description": "name of standard package type (imperial)",
+        "enum": [
```
Comment (rsnancollas) [inline part-spec/definitions.json:374]: examples, not enum

File: part-spec/definitions.json [hunk 6]
```diff
@@ -346,5 +346,41 @@
...
+        "description": "unit used to describe package dimensions",
+        "enum": [
+          "mils",
+          "milimeter",
+          "inches",
+          "centimeter"
+        ]
+      },
+      "standardPackageType": {
+        "description": "name of standard package type (imperial)",
```
Comment (rsnancollas) [inline part-spec/definitions.json]: standard package size

**Comment by rsnancollas** [comment]
Added a new commit - ready for review

File: part-spec/definitions.json [hunk 7]
```diff
@@ -346,5 +346,48 @@
...
+      },
+      "height": {
+        "description": "height of a package",
+        "type": "number"
+      },
+      "dimensionUnit": {
+        "description": "unit used to describe package dimensions",
+        "enum": [
+          "mils",
+          "milimeter"
```
Comment (asadate) [inline part-spec/definitions.json]: typo: millimeter

**Review by asadate — CHANGES_REQUESTED** [review]

**Comment by rsnancollas** [comment]
Typo fixed and ready for review

File: part-spec/definitions.json (hunk 7 above)
Comment (asadate) [inline part-spec/definitions.json]: Not sure why I can't see the change. It still shows "milimeter"

**Review by asadate — APPROVED** [review]

**Pull request merged**